website/docs/r/api_management.html.markdown

```diff
@@ -118,7 +118,11 @@ A `hostname_configuration` block supports the following:
 
 A `identity` block supports the following:
 
-* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this API Management Service. At this time the only supported value is`SystemAssigned`.
+~> **Note:** User Assigned Managed Identities are in Preview
+
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this API Management Service. Possible values are `SystemAssigned`, `UserAssigned` or `SystemAssigned, UserAssigned` (to enable both).
+
+* `identity_ids` - (Optional) A list of IDs for User Assigned Managed Identity resources to be assigned.
```

Comment:
Could we please add a note for this attribute to emphasize that this field must be set when type is `SystemAssigned` or `SystemAssigned, UserAssigned`? Something like this:
```
~> **NOTE:** This is required when `type` is set to `UserAssigned` or `SystemAssigned, UserAssigned`.
```

Reply:
Done in e8a1a04eb46c76f3c337ee9a5f54b8b9ae0f7a92.